libs/blocks/icon-block/icon-block.css

```diff
@@ -83,6 +83,10 @@
   gap: 16px;
 }
 
+.five-up .icon-block.full-width .foreground .text-content {
+  text-align: center;
+}
+
```

Comment:
Out of curiosity. Instead of adding this, why not remove the left align declaration[ from here](https://github.com/adobecom/milo/blob/0ba51c4aceeeca0dddc8d18acef150f92ab0b9ea/libs/blocks/icon-block/icon-block.css#L44)?  Then, the [center alignment here](https://github.com/adobecom/milo/blob/0ba51c4aceeeca0dddc8d18acef150f92ab0b9ea/libs/blocks/icon-block/icon-block.css#L33) kicks in.

Reply:
Thanks @rgclayton I realized the left alignment wasn't needed anyway... I would also rather remove it 👍🏻